recipes/fetch.md

```diff
@@ -12,20 +12,22 @@ fetch(API_URL)
 
 <Example flutter>
 
-```yaml
-dependencies:
-  http: ^0.12.0
-```
 
 ```dart
-import 'dart:convert' show json;
-import 'package:http/http.dart' as http;
-
-http.get(API_URL).then((http.Response res) {
-    final data = json.decode(res.body);
-    print(data);
-});
-
+import 'dart:io';
```

Comment:
why? the main purpose is to keep examples as simple as possible, your rewrite seem to be more complicated

Reply:
I think the best solution is adding both variants, because it is very important to know how to do some things without any dependencies, that can contain too much litter in many cases.